assets/src/scripts/store/index.spec.js

```diff
@@ -836,6 +876,8 @@ describe('Redux store', () => {
     });
 });
 
+const MOCK_WEATHER_SERVICE_DATA = '{"properties" : {"timeZone" : "America/Chicago"}}';
```

Comment:
I think it would be fine for this to go in the actual test suite for retrieveLocationTimeZone. It's not long and I think it should be the exception to separate test data from the test suite.